docs/freebsd/README.md

```diff
@@ -61,8 +64,21 @@ autoboot_delay="-1"
 console="comconsole"
 __EOF__
 ```
-Fetch a copy of the FreeBSD kernel sources and place them in `/usr/src`.  For instance, to get a copy of the current development sources, run:
-
+After VM is booted, /etc/rc.d/growfs should haven grown its file system automatically. Otherwise run:
+```console
+# /etc/rc.d/growfs onestart
+```
+The best way to build the custom kernel for SKYZKALLER is from the host.
+Shutdown the VM. <br>
+On the host:
+```console
+# mdconfig -a -f $IMAGEFILE
+# mount /dev/md0p4 /mnt
+# cp /etc/resolv.conf /mnt/etc
+# mount -t devfs none /mnt/dev
```

Comment:
You don't need to copy resolv.conf or mount devfs anymore, since this mount point is not being used for chroot.